File: Task-3/Python/Sub-Task4.py
def pyramid(rows):
    result = ""
    for i in range(0, rows, 2):
        result += " " * (rows - i - 1) + "* " * (i + 1) + "\n"
    for j in range(rows - 2, 0, -2):
        result += " " * (rows - j) + "* " * j + "\n"
    return result

with open("input.txt", 'r') as file:
    rows = int(file.read().strip()) 
with open("output.txt", 'w') as file2:
    file2.write(pyramid(rows))

File: Task-3/Rust/Sub-Task-4.rs
use std::fs::File;
use std::io::{self, Read, Write};

fn main() {
    let mut input_file = File::create("input.txt").unwrap(); 
    input_file.write_all(b"5").unwrap(); 
    let mut input_file = File::open("input.txt").unwrap(); 
    let mut input = String::new();
    input_file.read_to_string(&mut input).unwrap(); 
    let rows: usize = input.trim().parse().unwrap(); 
    let mut result = String::new();
    for i in (0..rows).step_by(2) {
        for _ in 0..(rows - i - 1) {
            result.push(' ');
        }
        for _ in 0..(i + 1) {
            result.push_str("* ");
        }
        result.push('\n');
    }
    for i in (0..(rows - 1)).step_by(2) {
        for _ in 0..(i + 2) {
            result.push(' ');
        }
        for _ in 0..(rows - i - 2) {
            result.push_str("* ");
        }
        result.push('\n');
    }

    let mut output_file = File::create("output.txt").unwrap(); 
    output_file.write_all(result.as_bytes()).unwrap(); /
}
